src/modules/roc_core/target_posix/roc_core/secure_random.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_core/target_posix/roc_core/secure_random.h
+//! @brief Helpers to work with CSPRNG.
+
+#ifndef ROC_CORE_SECURE_RANDOM_H_
+#define ROC_CORE_SECURE_RANDOM_H_
+
+#include "roc_core/stddefs.h"
+
+namespace roc {
+namespace core {
+
+//! Get secure random integer.
+//! @returns secure random value in range [from; to].
+uint32_t secure_random(uint32_t from, uint32_t to);
```

Comment:
I'd rephrase it from "secure random integer" to something like "random integer from CSPRNG". It's also worth to mention that it can fallback to usual PRNG on older libuv versions. And that it may block for a while.